3rd-party/submodule_info.md

```diff
@@ -7,41 +7,51 @@ info on where to find release information about the submodule.
 **This file needs updating when a submodule is updated!**
 
 ### gRPC
-Version: 1.25.0 (+[our patches](https://github.com/CanonicalLtd/grpc/compare/v1.25.0..3efaade))
-[https://github.com/CanonicalLtd/grpc.git](https://github.com/CanonicalLtd/grpc.git)
-[https://github.com/grpc/grpc/releases](https://github.com/grpc/grpc/releases)
+Version: 1.25.0 (+[our patches](https://github.com/CanonicalLtd/grpc/compare/v1.25.0..3efaade)) |
```

Comment:
This file wasn't in place when we updated gRPC recently.  Could you please update this to reflect that we are using version 1.30.2 plus our patches?  Thanks!

Reply:
Of course, will do!

Reply:
Done. I had to push the tags to our fork.